recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: bzip2
+  version: {{ version }}
+
+source:
+  fn: bzip2-{{ version }}.tar.gz
+  url: http://www.bzip.org/{{ version }}/bzip2-{{ version }}.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+
+build:
+  number: 0
```

Comment:
Should add a `skip` for Python 3.6 on Windows so as to avoid building the same thing twice.

```yaml
skip: true  # [win and py36]
```